src/index.js

```diff
@@ -137,12 +138,17 @@ module.exports = {
     },
 
     _getCapabilitiesFromConfig () {
-        const configPath = process.env.BROWSERSTACK_CAPABILITIES_CONFIG_PATH;
+        let relativeConfigPath;
 
-        if (!configPath)
+        if (process.env.BROWSERSTACK_CAPABILITIES_CONFIG_PATH) {
+            const configPath = path.resolve(process.env.BROWSERSTACK_CAPABILITIES_CONFIG_PATH);
+
+            relativeConfigPath = path.relative(__dirname, configPath);
```

Comment:
Since `path.resolve` already returns an absolute path, converting it to a relative path is unnecessary.